neurokit2/signal/signal_detrend.py

```diff
@@ -44,15 +45,19 @@ def signal_detrend(
         Only used if ``method`` is "loess". The parameter which controls the degree of smoothing.
     window : float
         Only used if ``method`` is "locreg". The detrending ``window`` should correspond to the
-        desired low frequency band to remove multiplied by the sampling rate
-        (for instance, ``1.5*1000`` will remove frequencies below 1.5Hz for a signal sampled at
-        1000Hz).
+        desired low frequency band to remove multiplied by the sampling rate if sampling_rate is not
+        provided to the function directly (for instance, ``1.5*1000`` will remove frequencies below 
+        1.5Hz for a signal sampled at 1000Hz).
     stepsize : float
         Only used if ``method`` is ``"locreg"``. Similarly to ``window``, ``stepsize`` should also
-        be multiplied by the sampling rate.
+        be multiplied by the sampling rate if sampling_rate is not provided.
```

Comment:
if ~not~ provided?

Reply:
I just left it out completely now, feel free to change if you think it should be specified